test/nodes/test_connector.py

```diff
@@ -18,7 +18,7 @@
 
 @pytest.fixture(scope="session")
 def test_url():
-    return f"file://{SAMPLES_PATH.absolute()}/crawler"
+    return (SAMPLES_PATH / "crawler").absolute().as_uri()
```

Comment:
Great to have pathlib handling the `/` for unix/windows automatically. 👍 Alternative would have been `SAMPLES_PATH.joinpath("crawler").absolute().as_uri()`